addons/fleet/tests/test_access_rights.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests import common, new_test_user
+
+
+class TestFleet(common.SavepointCase):
+
+    def test_manager_create_vehicle(self):
+        manager = new_test_user(self.env, "test fleet manager", groups="fleet.fleet_group_manager,base.group_partner_manager")
+        user = new_test_user(self.env, "test base user", groups="base.group_user")
+        brand = self.env["fleet.vehicle.model.brand"].create({
+            "name": "Audi",
+        })
+        model = self.env["fleet.vehicle.model"].create({
+            "brand_id": brand.id,
+            "name": "A3",
+        })
+        self.env["fleet.vehicle"].with_context(caca=True).with_user(manager).create({
```

Comment:
You can remove this context key here. It is a leftover I used for debugging purposes ^^
(note: I did not check the patch itself)